packages/renderer/src/lib/container/ContainerDetails.svelte

```diff
@@ -58,6 +60,9 @@ onMount(() => {
             router.goto(`${currentRouterPath}logs`);
           }
         }
+
+        const base64RepoTag = containerUtils.encodeContainerImage(container.image);
+        containerImageHref = `/images/${inspect.Image}/${container.engineId}/${base64RepoTag}/summary`;
```

Comment:
if we need that information, it should be handle on the 'main side'/server side as a field of the container object

Reply:
Okay, thank you for your feedback!